### PR TITLE
test: run ci on node 7,8

### PR DESCRIPTION
**Pull request by dead-horse**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,8 @@
...
 node_js:
-  - stable
-  - 7.6
-matrix:
-  include:
-    - node_js: "8.0"
-      env: "NVM_NODEJS_ORG_MIRROR=https://nodejs.org/download/nightly"
-  allow_failures:
-    # Allow the nightly installs to fail
-    - env: "NVM_NODEJS_ORG_MIRROR=https://nodejs.org/download/nightly"
```
Comment (iyuq) [inline .travis.yml]: I think node 6.x should be tested.

Comment (fl0w) [inline .travis.yml]: Koa 2 is communicated as node >7.6 only.

Comment (iyuq) [inline .travis.yml]: @fl0w Should‘t remove v6 until 8 become LTS?

Comment (fl0w) [inline .travis.yml]: There's no point in testing and actively developing against node 6 if 7.6> is the target. There's however nothing working against a user to themselves transpile Koa 2 down to node 6.

Comment (jonathanong) [inline .travis.yml]: i'm okay with testing node 6 as long as it doesn't require work (i.e. babelfying everything)

**Pull request merged**